hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/SnapshotDiffJob.java

```diff
@@ -269,9 +269,11 @@ public SnapshotDiffJobProto toProtoBuf() {
         .setForceFullDiff(forceFullDiff)
         .setDisableNativeDiff(disableNativeDiff)
         .setTotalDiffEntries(totalDiffEntries)
-        .setSubStatus(subStatus.toProtoBuf())
-        .setKeysProcessedPct(keysProcessedPct)
-        .build();
+        .setKeysProcessedPct(keysProcessedPct);
+    if (subStatus != null) {
+      builder.setSubStatus(subStatus.toProtoBuf());
+    }
```

Comment:
If `subStatus` can be null, maybe we also need use `diffJobProto.hasSubStatus` before in `SnapshotDiffJob#getFromProtoBuf`.